tensorflow_addons/optimizers/lamb_optimizer_test.py

```diff
@@ -0,0 +1,461 @@
+
+                if not tf.executing_eagerly():
+                    update1 = opt.apply_gradients(
+                        zip([grads0, grads1], [var0, var1]))
+                    update2 = opt.apply_gradients(
+                        zip([grads0, grads1], [var0, var1]))
+                    self.evaluate(tf.compat.v1.global_variables_initializer())
+
+                # Fetch params to validate initial values
+                self.assertAllClose([1.0, 2.0], self.evaluate(var0))
+                self.assertAllClose([3.0, 4.0], self.evaluate(var1))
+
+                # Run 3 steps of intertwined LAMB1 and LAMB2.
+                for t in range(3):
+                    beta_1_power, beta_2_power = get_beta_accumulators(
+                        opt, dtype)
+                    self.assertAllCloseAccordingToType(
+                        0.9**(t + 1), self.evaluate(beta_1_power))
+                    self.assertAllCloseAccordingToType(
+                        0.999**(t + 1), self.evaluate(beta_2_power))
+
+                    if not tf.executing_eagerly():
+                        if t % 2 == 0:
+                            update1.run()
+                        else:
+                            update2.run()
+                    else:
+                        opt.apply_gradients(
+                            zip([grads0, grads1], [var0, var1]))
+
+                    var0_np, m0, v0 = lamb_update_numpy(
+                        var0_np, grads0_np, t, m0, v0)
+                    var1_np, m1, v1 = lamb_update_numpy(
+                        var1_np, grads1_np, t, m1, v1)
+
+                    # Validate updated params
+                    self.assertAllCloseAccordingToType(var0_np,
+                                                       self.evaluate(var0))
+                    self.assertAllCloseAccordingToType(var1_np,
+                                                       self.evaluate(var1))
+
+    @test_utils.run_in_graph_and_eager_modes(use_gpu=False)
+    def testMinimizeMeanSquareLossWithWeightDecay(self):
+        with self.cached_session():
+            w = tf.Variable([0.1, -0.2, -0.1])
+            x = tf.constant([0.4, 0.2, -0.5])
+            loss = lambda: tf.reduce_mean(tf.square(x - w))  # pylint:disable=cell-var-from-loop
+            opt = lamb_optimizer.LAMBOptimizer(0.02, weight_decay_rate=0.01)
+
+            if not tf.executing_eagerly():
+                op = opt.minimize(loss, [w])
+                self.evaluate(tf.compat.v1.global_variables_initializer())
+
+            self.evaluate(tf.compat.v1.global_variables_initializer())
+            # Run 200 steps
+            for _ in range(200):
+                if tf.executing_eagerly():
+                    opt.minimize(loss, [w])
+                else:
+                    self.evaluate(op)
+            # Validate updated params
+            self.assertAllClose(
+                self.evaluate(w), [0.4, 0.2, -0.5], rtol=1e-2, atol=1e-2)
+
+
```

Comment:
Can you add a quick test to verify the config for serialization. See:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/rectified_adam_test.py#L182